app/javascript/components/lease_ups/LeaseUpApplicationsFilter.js

```diff
@@ -39,6 +44,9 @@ const LeaseUpApplicationsFilter = ({ onSubmit, preferences = [], loading = false
                 <div className='filter-group_item'>
                   <SelectField fieldName='status' options={StatusOptions} placeholder='Status' />
                 </div>
+                <div className='filter-group_item'>
+                  <SelectField fieldName='accessibility' options={accessibilitFilterOptions} placeholder='Accessibility Requests' />
```

Comment:
Typo in accessibilityOptions both above and here
```suggestion
                  <SelectField fieldName='accessibility' options={accessibilityFilterOptions} placeholder='Accessibility Requests' />
```